README.md

```diff
@@ -115,6 +115,7 @@ config object:
 |-------------|---------------------|-------------|----------------------|----------|--------------|
 | `zip`       | All                 | Zips your packaged application | None | Yes | `zip` on Darwin/Linux |
 | `squirrel`  | Windows             | Generates an installer and `.nupkg` files for Squirrel.Windows | [`electronWinstallerConfig`](https://github.com/electron/windows-installer#usage) | Yes |  |
+| `appx`      | Windows             | Generates a Windows Store package | [`windowsStoreConfig`](https://github.com/felixrieseberg/electron-windows-store#programmatic-usage) | No |  |
```

Comment:
Per one of your commit messages, do we want to mention that this requires the Windows SDK?